src/composables/sdk.ts

```diff
@@ -0,0 +1,49 @@
+import type {
+  IDefaultComposableOptions,
+  Protocol,
+} from '../types';
+import { useAccounts } from './accounts';
+import { IChainProtocol } from '../types';
+import { AeternityProtocol } from '../protocols/aeternity/AeternityProtocol';
+import { BitcoinProtocol } from '../protocols/bitcoin/BitcoinProtocol';
+import { DogecoinProtocol } from '../protocols/dogecoin/DogecoinProtocol';
+import {
+  PROTOCOL_AETERNITY,
+  PROTOCOL_BITCOIN,
+  PROTOCOL_DOGECOIN,
+} from '../popup/utils/constants';
+
+let isSdkRegistryReady = false;
+let sdkRegistry: Record<Protocol, IChainProtocol>;
+
+export function useSdk({ store }: IDefaultComposableOptions) {
+  const { activeAccount } = useAccounts({ store });
+
+  if (!isSdkRegistryReady) {
+    sdkRegistry = {
+      [PROTOCOL_AETERNITY]: new AeternityProtocol(store),
+      [PROTOCOL_BITCOIN]: new BitcoinProtocol(),
+      [PROTOCOL_DOGECOIN]: new DogecoinProtocol(),
+    };
+  }
+  isSdkRegistryReady = true;
```

Comment:
I still think this not good to initialize all classes whenever the `useSdk` composable is fired. What about situations where user opens a component, which uses this composable, but user didn't take any action that requires any of the protocol SDKs?

I think the libs should be lazy loaded only when needed.

Reply:
Sure. I will add the `initSdk` method as part of `getSdk` method